modules/liftover/1.0/config/default.yaml

```diff
@@ -24,7 +24,7 @@ lcr-modules:
             bedhg38tobedhg19: "{MODSDIR}/src/chains/hg38ToHg19.over.chain"
             bedhg19toseghg19: "{MODSDIR}/src/convert_for_liftover.py"
         conda_envs:
-            liftover-366: "{MODSDIR}/envs/liftover-366.yaml"
+            liftover-366: "{MODSDIR}/envs/liftover-377.yaml"
```

Comment:
I'm guessing this would need to be changed in the module, and that's not really the purpose of this PR for now, so I will create an issue and suggest that the key should be version-agnostic.